src/deadline_test_fixtures/deadline/resources.py

```diff
@@ -736,7 +736,7 @@ def assert_single_task_log_contains(
         logs_client: BaseClient,
         expected_pattern: re.Pattern | str,
         assert_fail_msg: str = "Expected message not found in session log",
-        retries: int = 4,
+        retries: int = 6,
```

Comment:
Changing the default value for an optional argument is technically a breaking change based on our [versioning policy](https://github.com/aws-deadline/deadline-cloud-test-fixtures?tab=readme-ov-file#versioning).

We have two paths forward as I see it:

1.  Mark this as a breaking change using [conventional commit syntax](https://www.conventionalcommits.org/en/v1.0.0/)
2.  Since this is to fix an issue in the worker agent tests, we can provide an alternate value for `retries` in the affected tests there

What do you think?

Reply:
I think breaking change makes sense. Since we are constantly getting longer than expected cloudwatch logs treaming times, it would make sense to change this util's number of retries across all functions. I will mark this as a breaking change if that's alright with you